packages/Algernon.package/ALGNamedWorldMenuEntry.class/instance/allEntries.st

```diff
@@ -0,0 +1,10 @@
+accessing
+allEntries
```

Comment:
`allEntries` exists on `ALGNamedWorldMenuEntry` and `ALGWorldMenuType` - are both necessary?

Reply:
Yes, one scans the docking bar entries (submorphs of docking bar) and the other one the submenu entries (submorphs of submenu of menu entry)
There is very little difference but quite a bit of duplication, so I'm not really satisfied with it either. But then again, SWT taught me to only deduplicate whenever I do the same thing for the **3rd** time 🙈

Reply:
this was really hard to grasp when laura and I started working last week, maybe we could add a short comment that outlines the situation or just deduplicate? :smile:

I think that a general helper method for traversing / collecting the submorphs on the one that is not an `ALGType` would be great, you can decide though